src/main/java/seedu/address/logic/commands/getcommands/GetAppointmentByDateCommand.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.commands.getcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.GetCommand;
+import seedu.address.model.Model;
+import seedu.address.model.person.AppointmentByDatePredicate;
+
+/**
+ * Finds and lists all patients on the same appointment date.
+ *
+ */
+public class GetAppointmentByDateCommand extends GetCommand {
+
+    public static final String APPOINTMENT_BY_DATE_PREFIX = "/apptbd";
```

Comment:
Would it be possible to change this prefix to `/appton`? Sounds better and is easier for users to read.